src/common/com/intellij/plugins/haxe/model/type/HaxeTypeCompatible.java

```diff
@@ -29,6 +29,19 @@ static public boolean canApplyBinaryOperator(SpecificTypeReference a, SpecificTy
     return true;
   }
 
+  static public boolean isAssignable(SpecificTypeReference a, ResultHolder b) {
```

Comment:
In a static context I usually use isAssignableFromTo and in a non-static context I would use isAssignableFrom or isAssignableTo. Indeed a and b are not good names. They are sort and provides ordering information a is before b. But I understand that they don't read good, so I will change them and will change the method name too.